frontend/ui/widget/imageviewer.lua

```diff
@@ -431,6 +425,7 @@ function ImageViewer:_new_image_wg()
         file = self.file,
         image = self.image,
         image_disposable = false, -- we may re-use self.image
+        file_do_cache = self.image and true or false, -- do not cache local files
```

Comment:
Is it still valuable to keep having it true with image data?
(If I didn't touch the default of true, may be it's just because I didn't think about it :) Feels like with ImageViewer, and one-time looking at images, whether data or local file, we don't need to cache anything?)